roofit/histfactory/inc/RooStats/HistFactory/Data.h

```diff
@@ -27,10 +27,12 @@ class Data {
   //friend class Channel;
 
   Data();
-  Data( const Data& other );
+  Data( const Data& other ) = default;
```

Comment:
Can we not simply remove the declarations of copy ctor and assignment op?

Reply:
Sure, sometimes you don't see the simple things.

Reply:
I?! ;-)))